apps/passport-client/src/appHooks.ts

```diff
@@ -1,10 +1,13 @@
+import { useContext, useEffect, useState } from "react";
+
```

Comment:
you've got many files in which you added extra whitespace into the imports section, plz fix that across all the files